src/content/docs/infrastructure/prometheus-integrations/install-configure-remote-write/set-your-prometheus-remote-write-integration.mdx

```diff
@@ -34,18 +34,29 @@ Go to the [Prometheus remote write setup launcher in New Relic One](https://one.
 
    Use the following syntax:
 
+   **Prometheus v2.26 and newer**
+   ``` 
+   remote_write: 
+   - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=<var>YOUR_DATA_SOURCE_NAME</var>  
+     authorization:
+       credentials: YOUR_LICENSE_KEY
+   ```
+
+   **Prometheus older than v2.26**
    ```
    remote_write: 
    - url: https://metric-api.newrelic.com/prometheus/v1/write?prometheus_server=<var>YOUR_DATA_SOURCE_NAME</var>  
      bearer_token:<var>YOUR_LICENSE_KEY</var>
    ```
-
-   OR
-
+   
+   OR 
+   
+   **Any Prometheus version**
    ```
    remote_write: 
    - url: https://metric-api.newrelic.com/prometheus/v1/write?X-License-Key=<var>YOUR_LICENSE_KEY</var>&prometheus_server=<var>YOUR_DATA_SOURCE_NAME</var>
    ```
+   *This approach is discouraged as it is preferable not to pass credentials in the url*
```

Comment:
@cmacnaug I'd love to use something with a little less passive voice.

How about: "This approach passes credentials in the URL. We don't recommend using it."?

Reply:
That sounds fine to me. I'm honestly not sure that we still have a use case for this configuration mode, but didn't want to remove it in case it was useful for a user. 

"This approach passes credentials in the URL. We don't recommend using it unless one of the above approaches doesn't work in your environment."

Reply:
No, I think that's fine. I'd change it slightly to:

"This approach passes credentials in the URL. We don't recommend using it unless one of these other approaches doesn't work in your environment." 

What do you think @cmacnaug?

Reply:
Works for me, Thanks. I think based on Tyler's testing we should be good to go with these changes.